release-notes/3.0/preview/3.0.0-preview-known-issues.md

```diff
@@ -2,6 +2,19 @@
 
 This document lists known issues for **.NET Core 3.0 Preview releases** which may be encountered during usage.
 
+## .NET Core
+
+### Preview 5
+
+- **WinForms not showing correctly when publishing with PublishReadyToRun=true**
```

Comment:
We found VisualStyle as the major issue. Can we limit it to that? this statement is so broad and if there is any other issue, they might just try to categorize it to this issue.

Reply:
Sure. I can put `Visual style in WinForms app not showing correctly`. Or is there a better way you'd like to word it?